packages/core/dist/js-joda.d.ts

```diff
@@ -1549,6 +1550,183 @@ export class YearMonth extends Temporal implements TemporalAdjuster {
+    static of(dateTime: LocalDateTime, offset: ZoneOffset): OffsetDateTime;
+    static of(date: LocalDate, time: LocalTime, offset: ZoneOffset): OffsetDateTime;
+    static of(year: number, month: number, day: number, hour: number, minute: number, second: number, nanoOfSecond: number, offset: ZoneOffset): OffsetDateTime;
+    static ofInstant(instant: Instant, zone: ZoneId): OffsetDateTime;
+    static parse(text: string, formatter?: DateTimeFormatter): OffsetDateTime;
+
+    private constructor();
+
+    adjustInto(temporal: Temporal): Temporal
+    atZoneSameInstant(zone: ZoneId): ZonedDateTime
+    atZoneSimilarLocal(zone: ZoneId): ZonedDateTime
+    compareTo(other: OffsetDateTime): number
+    equals(obj: any): boolean
+    format(formatter: DateTimeFormatter): string
+    get(field: TemporalField): number
+    dayOfMonth(): number
+    dayOfWeek(): DayOfWeek
+    dayOfYear(): number
+    hour(): number
+    getLong(field: TemporalField): number
+    minute(): number
+    month(): Month
+    monthValue(): number
+    nano(): number
+    offset(): ZoneOffset
+    second(): number
+    year(): number
+    hashCode(): number
+    isAfter(other: OffsetDateTime): boolean
+    isBefore(other: OffsetDateTime): boolean
+    isEqual(other: OffsetDateTime): boolean
+    isSupported(fieldOrUnit: TemporalField | TemporalUnit): boolean
+    minus(amountToSubtract: number, unit: TemporalUnit): OffsetDateTime
+    minus(amountToSubtract: TemporalAmount): OffsetDateTime
+    minusDays(days: number): OffsetDateTime
+    minusHours(hours: number): OffsetDateTime
+    minusMinutes(minutes: number): OffsetDateTime
+    minusMonths(months: number): OffsetDateTime
+    minusNanos(nanos: number): OffsetDateTime
+    minusSeconds(seconds: number): OffsetDateTime
+    minusWeeks(weeks: number): OffsetDateTime
+    minusYears(years: number): OffsetDateTime
+    plus(amountToAdd: number, unit: TemporalUnit): OffsetDateTime
+    plus(amountToAdd: TemporalAmount): OffsetDateTime
+    plusDays(days: number): OffsetDateTime
+    plusHours(hours: number): OffsetDateTime
+    plusMinutes(minutes: number): OffsetDateTime
+    plusMonths(months: number): OffsetDateTime
+    plusNanos(nanos: number): OffsetDateTime
+    plusSeconds(seconds: number): OffsetDateTime
+    plusWeeks(weeks: number): OffsetDateTime
+    plusYears(years: number): OffsetDateTime
+    query<T>(query: TemporalQuery<T>): T
+    range(field: TemporalField): ValueRange
+    toEpochSecond(): number
+    toJSON(): string;
+    toInstant(): Instant
+    toLocalDate(): LocalDate
+    toLocalDateTime(): LocalDateTime
+    toLocalTime(): LocalTime
+    toOffsetTime(): OffsetTime
+    toString(): string
+    toOffsetDateTime(): OffsetDateTime
+    truncatedTo(unit: TemporalUnit): OffsetDateTime
+    until(endExclusive: Temporal, unit: TemporalUnit): number
+    with(adjuster: TemporalAdjuster): OffsetDateTime
+    with(field: TemporalField, newValue: number): OffsetDateTime
```

Comment:
Unfortunately implementation of `with` is still missing. must be implemented.

Reply:
removed

Reply:
@pithu `with` should be inherited from class `Temporal`:

https://github.com/js-joda/js-joda/blob/166f123d92ccd9d4ebf1ef86fe992e6c233791aa/packages/core/src/temporal/Temporal.js#L291-L297

Isn't it? `OffsetDateTime` and `OffsetTime` both extends that class.

Reply:
Yes and it's exercised in the tests